mlflow/ai_commands/genai/analyze_experiment.md

```diff
@@ -0,0 +1,216 @@
+---
+namespace: genai
+description: Analyzes the traces logged in an MLflow experiment to find operational and quality issues automatically, generating a markdown report.
+---
+
+# Analyze Experiment
+
+Analyzes traces in an MLflow experiment for quality issues, performance problems, and patterns.
+
+## Step 1: Setup and Configuration
+
+### 1.1 Collect Experiment Information
+
+- **REQUIRED FIRST**: Ask user "How do you want to authenticate to MLflow?"
+
+  **Option 1: Local/Self-hosted MLflow**
+
+  - Ask for tracking URI (one of):
+    - SQLite: `sqlite:////path/to/mlflow.db`
+    - PostgreSQL: `postgresql://user:password@host:port/database`
+    - MySQL: `mysql://user:password@host:port/database`
+    - File Store: `file:///path/to/mlruns` or just `/path/to/mlruns`
+  - Set `MLFLOW_TRACKING_URI=<provided_uri>`
+
+  **Option 2: Databricks**
+
+  - Ask which authentication method:
```

Comment:
Can it also get ENV VARs that have been set via bashrc / zshrc and read them directly?